QUICKSTART-EKS.md

```diff
@@ -105,6 +105,9 @@ Supported variants and architectures are described in the [README](README.md#var
 For the purposes of SSM parameters, the valid architecture names are `x86_64` and `arm64` (also known as `aarch64`).
 Also, if you know a specific Bottlerocket version you'd like to use, for example `1.11.0`, you can replace `latest` with that version.
 
+Our EKS variants with NVIDIA support append `-nvidia` to the variant name. For instance,
+the variant for Kubernetes version 1.28 with NVIDIA support is `aws-k8s-1.28-nvidia`.
```

Comment:
nit: The voice in this doc avoid possession for Bottlerocket. We also put each new sentence on a new line.
```suggestion
Bottlerocket EKS variants with NVIDIA support append `-nvidia` to the variant name. 
For instance, the variant for Kubernetes version 1.28 with NVIDIA support is `aws-k8s-1.28-nvidia`.
```